hydra-node/test/Hydra/API/ServerOutputSpec.hs

```diff
@@ -20,10 +20,17 @@ spec = parallel $ do
   roundtripAndGoldenSpecsWithSettings
     settings
     (Proxy @(ReasonablySized (ServerOutput SimpleTx)))
+
   roundtripAndGoldenSpecsWithSettings
     settings
     (Proxy @(ReasonablySized (ServerOutput Tx)))
 
+  -- NOTE: The golden file produced by this is also used by the
+  -- 'validate:outputs' target in ./docs/package.json.
```

Comment:
I feel we should document somewhere (maybe in CONTRIBUTING "Building & Testing" section)
the process of recreating golden files and how to validate the json-schema with them.

Reply:
We should at least mention the invocation how to check these things, i.e. `yarn validate` in `docs/` as a thing you might want to do as a acontributor.

Reply:
shall we add it as part of this PR or in a separate one?

which part from `docs/` does it make sense to update?